docs/app/pages/components/components-sections/dashboard/widgets/dashboard-widgets.component.html

```diff
@@ -0,0 +1,188 @@
+<ux-dashboard
+    #dashboard
+    class="customizable-dashboard"
+    [class.dashboard-grabbing]="dashboard.isGrabbing"
+    [options]="options">
+
+    <ux-dashboard-text-widget
+        class="widget"
+        id="widget-text"
+        name="Text Widget"
+        heading="Text Widget"
+        [fixedMode]="fixedMode"
+        [text]="loremIpsum"
+        [editable]="textWidgetEditable"
+        [colSpan]="2"
+        [rowSpan]="2">
+    </ux-dashboard-text-widget>
+
+    <ux-dashboard-table-widget
+        class="widget"
+        id="widget-table"
+        name="Table Widget"
+        heading="Table Widget"
+        [fixedMode]="fixedMode"
+        [header]="tableHeader"
+        [data]="tableData"
+        [colSpan]="2"
+        [rowSpan]="2">
+    </ux-dashboard-table-widget>
+
+    <ux-dashboard-actions-widget
+        class="widget"
+        id="widget-actions"
+        name="Actions Widget"
+        heading="Actions Widget"
+        [fixedMode]="fixedMode"
+        [status]="status"
+        [actions]="actions"
+        [colSpan]="2"
+        (actionSelected)="handleActionChange($event)">
+        <ng-template #iconAccept>
+            <span class="custom-icon">✅</span>
```

Comment:
This is actually the demonstration that any picture can be used in this location. In case you want to use a UX Aspects icon, you can specify it directly in the configuration (see docs/app/pages/components/components-sections/dashboard/widgets/dashboard-widgets.component.ts:82):

`            {value: 'accept', label: 'Accept', iconTemplate: this.iconAccept},
            {value: 'decline', label: 'Decline', icon: 'close'}
`

Reply:
Understood, however consumers very often copy/paste the example code, so I prefer the icon option in this case.

Reply:
I changed the example to contain a simple letter 'X' - like this, we can show the configurated icon without using the default, which is a UX icon.

Reply:
This is fine. Thanks.